src/PokemonType/GrassPokemon.java

```diff
@@ -0,0 +1,84 @@
+package PokemonType;
+
+import Abstract.Pokemon;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GrassPokemon extends Pokemon {
+    private String knockOff;
+    private String sludgeBomb;
+    private String gigaDrain;
+    private String synthesis;
+
+    //constructor
+    public GrassPokemon(String name, String knockOff, String sludgeBomb, String gigaDrain, String synthesis) {
+        super(name, new ArrayList<>(List.of("Grass", "Poison")));
```

Comment:
Hiermee zeg je dat alle grass types ook poison hebben,    ivysaur.addType("Poison"); In de main had dit bijvoorbeeld ook gedaan :)